requirements.txt

```diff
@@ -1,8 +1,8 @@
 attr==0.3.*
 inflect==2.1.*
 lmfit==0.9.*
-matplotlib>=3.0.*
-numpy==1.16.*
+matplotlib>==3.5.3*
+numpy==1.21.*
```

Comment:
From issue #91:

matplotlib 3.5.3 has requirement numpy>=1.17, but you'll have numpy 1.16.6 which is incompatible.

Cna we address this here?

Reply:
Yes, it's probably best if I take a bit to just update the crucial packages to newer vesions and if anything breaks I'll address if before pull requesting again